habitat_baselines/config/eqa/il_edfe.yaml

```diff
@@ -0,0 +1,28 @@
+BASE_TASK_CONFIG_PATH: "configs/tasks/eqa_mp3d.yaml"
+TRAINER_NAME: "edfe"
```

Comment:
I am not sure what EDFE stands for.  Can we just call this EQA?

Reply:
EDFE is an Encoder-Decoder based Feature Extractor - a pretrained version of which is used for extracting features from images for the VQA trainer, the NAV trainer (PACMAN) and the RL-based joint fine-tuning trainer (which I thought of as calling as the EQA trainer). I have provided more information about the role of the three trainers [here](https://github.com/mukulkhanna/habitat-api/blob/eqa-1/habitat_baselines/il/README.md).

In my understanding, a total of four trainers would be required - EDFE (feat extractor), VQA, NAV/PACMAN and the (to-be created) EQA trainer.

Reply:
If I was a person looking at this for the first time, and I saw EDFE I would have no idea what that is or that it is related to EQA.  What about `eqa-cnn-pretrain`?  Or `semseg-ae-depth-cnn-pretrain` if you want to decouple it from EQA.

Reply:
I'll go with `eqa-cnn-pretrain` then. Thanks